src/main/java/techcourse/myblog/web/controller/UserControllerExceptionHandler.java

```diff
@@ -0,0 +1,49 @@
+package techcourse.myblog.web.controller;
+
+import org.springframework.validation.BindException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import org.springframework.web.servlet.view.RedirectView;
+import techcourse.myblog.service.dto.UserDto;
+import techcourse.myblog.support.exception.*;
+
+@ControllerAdvice
+public class UserControllerExceptionHandler {
+    @ExceptionHandler(LoginFailedException.class)
+    public RedirectView handleLoginFail(LoginFailedException e, RedirectAttributes redirectAttributes) {
+        redirectAttributes.addFlashAttribute("error", e.getMessage());
+        return new RedirectView("/login");
+    }
+
+    @ExceptionHandler(DuplicatedEmailException.class)
+    public RedirectView handleSignupFail(DuplicatedEmailException e, RedirectAttributes redirectAttributes) {
+        redirectAttributes.addFlashAttribute("error", e.getMessage());
+        return new RedirectView("/signup");
+    }
+
+    @ExceptionHandler(MismatchAuthorException.class)
+    public RedirectView handleMismatchArticleAuthorFail(MismatchAuthorException e, RedirectAttributes redirectAttributes) {
+        redirectAttributes.addFlashAttribute("error", e.getMessage());
+        return new RedirectView("/");
+    }
+
+    @ExceptionHandler(NotFoundArticleException.class)
```

Comment:
MismatchAuthorException 처리와 같은 것 같네요!
아래와 같이 사용해보면 어떨까요?

```suggestion
    @ExceptionHandler({NotFoundArticleException.class, MismatchAuthorException.class})
```